refinery/core/views.py

```diff
@@ -774,25 +774,26 @@ def get(self, request):
             request.user, 'dataset'
         ).order_by('-modification_date')
 
-        total_data_sets = len(user_data_sets)
-        if filters.get('is_owner') or filters.get('is_public') or \
-                filters.get('group'):
-            filtered_data_set = []
-            for data_set in user_data_sets:
-                if not data_set.is_valid:
-                    logger.warning(
-                        "DataSet with UUID: {} is invalid, and most likely is "
-                        "still being created".format(data_set.uuid)
-                    )
+        filtered_data_sets = []
+        filter_requested = filters.get('is_owner') \
+            or filters.get('is_public') \
+            or filters.get('group')
+        for data_set in user_data_sets:
+            if not data_set.is_valid:
+                logger.warning(
+                    "DataSet with UUID: {} is invalid, and most likely is "
+                    "still being created".format(data_set.uuid)
+                )
```

Comment:
I think a `continue` after this logger would be ideal

Reply:
@scottx611x Why?

Reply:
I guess it won't change any functionality right now, but its a good visual indicator that the current iteration is complete. If any logic was ever added under the `else`, but still within the loop, it would be hit for a DataSet that is invalid without it.